packages/liquidator/src/liquidator.js

```diff
@@ -440,11 +454,16 @@ class Liquidator {
         gas: Math.min(Math.floor(gasEstimation * this.GAS_LIMIT_BUFFER), this.txnGasLimit),
         gasPrice: this.gasEstimator.getCurrentFastPrice()
       };
+      // In contract version 1.2.2 and below this function returns one value: the amount withdrawn by the function caller.
```

Comment:
Since versions are linearly increasing, and some features don't change "greater than" or "less than" certain versions, is there an easy way to replicate the logic `if (this.contractVersion <= 1.2.2)`...?

Reply:
yup, we could build something like that out. would need to create an override for the comparison operator or a function to enable us to do this. For now, though explicit version matching is easiest I think.